lib/orderbook/OrderBookRepository.ts

```diff
@@ -40,6 +44,24 @@ class OrderbookRepository {
       await this.models.Order.create(order as OrderAttributes);
     }
   }
+
+  /**
+   * Only persist multiple orders if they don't exists.
+   * @param orders array of orders to persist
+   */
+  public addOrdersIfNotExists = async (orders: db.OrderFactory[]) => {
+    const ordersToAdd: Promise<void>[] = [];
+
+    await Bluebird.Promise.each(orders, (order) => {
```

Comment:
I'm not exactly sure what `Bluebird.Promise.each` does, but I think you can just do `orders.forEach` without needing `await`. It might wind up doing the same thing but it's less verbose at least.